### PR TITLE
DM-125 Feature/jesse/org form

### PR DESCRIPTION
**Pull request by jjbandit**
I rebased to fix up some commit history so I figured I'd create a new branch instead of force-overwriting the error handling one.  I also ended up doing more than just error handling, so I think that makes sense to rename it.

### CODE REVIEW

File: src/front-end/typescript/lib/pages/organization/components/form.tsx [hunk 1]
```diff
@@ -33,34 +38,41 @@ export interface State {
...
+  | ADT<'contactPhone',    ShortText.Msg>
+  | ADT<'submit',          SubmitHook | undefined>
+  | ADT<'onChangeAvatar',  File>
+  | ADT<'region',          ShortText.Msg>
   ;
 
 export type Msg = GlobalComponentMsg<InnerMsg, Route>;
 
-export type Values = Required<CreateRequestBody>;
+export type Values = Required<CreateRequestBody> & { newLogoImage?: File };
```
[the remark this answers is not in the document]
Comment (dhruvio) [reply src/front-end/typescript/lib/pages/organization/components/form.tsx]: Ignore the previous comment in this thread, we should keep `setValues`.

File: src/front-end/typescript/lib/pages/organization/create.tsx [hunk 2]
```diff
@@ -83,10 +83,9 @@ const view: ComponentView<State, Msg> = ({ state, dispatch }) => {
       <Row>
         <Col xs='12'>
           <OrgForm.view
-            submitHook={() => {}}
             state={state.govProfile}
             disabled={false}
-            dispatch={mapComponentDispatch(dispatch, value => adt('govProfile' as const, value))} />
+            dispatch={mapGlobalComponentDispatch(dispatch, value => adt('govProfile' as const, value))} />
```
Comment (dhruvio) [inline src/front-end/typescript/lib/pages/organization/create.tsx]: Successfully creating an organization should redirect to `userProfile` where `tab = 'organizations'`

**Comment by dhruvio** [comment]
Added another comment. I think that's it for the review, let me know once you've pushed your changes 👍

**Comment by jjbandit** [comment]
I reviewed and implemented _most_ changes, with the exception of the following:

1) DM-182 needs to be complete before we can redirect to the userProfile page, search for tag '@org-needs-owner-information'

2) There were a bunch of 'remove XXXX class' or 'add XXXX class' comments which didn't give me a clear idea of what the overarching issue(s) were.  I looked closely at the design and userProfile page to fix the alignment issues.  In the future for styling-related issues let's either just fix them instead of writing a comment, or lump them all into a 'the grid alignment for page X'  or 'component Y looks off ...' comment should be sufficient.

3) I noticed there's a cancel button on the Create page .. should we be doing a redirect there or just clearing the form?  Or should that button even exist..?

**Pull request merged**